executor/src/witgen/global_constraints.rs

```diff
@@ -102,15 +102,17 @@ impl<T: FieldElement> RangeConstraintSet<&AlgebraicReference, T> for GlobalConst
     }
 }
 
+type SelExpressions<T> = SelectedExpressions<Expression<T>>;
```

Comment:
I think in witgen, there should be something like
`type Identity<T> = Identity<SelectedExpressions<Expression<T>>>`

Reply:
Maybe this even makes sense in some parts of analyzed...